generators/generator-docker-base.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2013-2018 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see https://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const chalk = require('chalk');
+const prompts = require('./docker-prompts');
+const BaseGenerator = require('./generator-base');
+const { loadFromYoRc, checkDocker, checkImages, generateJwtSecret, setAppsFolderPaths } = require('./docker-base');
+const statistics = require('./statistics');
+
+const constants = require('./generator-constants');
+
+module.exports = class extends BaseGenerator {
```

Comment:
You've named this file `generator-docker-base.js`, and I'm OK with it.
But I recently used `generator-base-blueprint.js` for the base generator for blueprint. For the sake of consistency, I think I should rename the blueprint one to `generator-blueprint-base.js`, shouldn't I?

Reply:
no worries i'll rename to `generator-base-docker.js`, I think thats more uniform